pipeline/interpolate_gene.py

```diff
@@ -182,6 +182,10 @@ def main(
         )
         predicted_volume = gene_optical_flow.predict_volume()
 
+    # Mirror the volume if the dataset is sagittal
+    if axis == "sagittal":
+        predicted_volume[:, :, 228:] = np.flip(predicted_volume[:, :, :228], axis=2)
```

Comment:
I know that currently the pipeline only works on the `(528, 320, 456)` volumes. However, we might extend it 
in the future. Could we maybe turn this `456` into a variable and then just ` // 2` so that the code becomes more maintainable?

Reply:
Really good point! Done in [47596e0](https://github.com/BlueBrain/Deep-Atlas/pull/22/commits/47596e0f5f7ca44b87718dde4677bf982931283e).

Reply:
Btw the `predicted_volume` seems to be coming from `atlinter`. Is the shape hardcoded there?

Reply:
No it is not, when constructing `GeneDataset`, one can specify the shape of the volume (check [here](https://github.com/BlueBrain/atlas-interpolation/blob/c560a912f4e772340a92c6e0652bd9e23dd12ab8/src/atlinter/data.py#L47))

Reply:
Ahh, pretty cool! Great! Thank you!